packages/plugins/input-exercise/src/editor.tsx

```diff
@@ -48,6 +48,18 @@ export function InputExerciseEditor(props: InputExerciseProps) {
       </PreviewOverlay>
       {nestedFocus && !previewActive && (
         <React.Fragment>
+          <div style={{ marginBottom: '0.5em' }}>
+            <OverlaySelect
```

Comment:
This shouldn't be used here. As the name suggests, `OverlaySelect` is intended to be used in the overlay, and may not look correctly in the editor itself (since the plugin toolbar stuff can be overridden). So please use your own components here (as done in the sc/mc component).

Reply:
Furthermore: to be consistent with sc/mc, you should add a separator after the label, too.

Reply:
separator = ": "